static/js/pm_list.js

```diff
@@ -62,8 +64,14 @@ function remove_expanded_private_messages() {
     resize.resize_stream_filters_container();
 }
 
+function open_private_more_messages() {
+    $(".expanded_private_messages").expectOne().removeClass("zoom-out").addClass("zoom-in");
```

Comment:
removeClass is actually not needed. I've not ever seen `zoom-out` there. `zoom-out` is added to `li`elem above `.global-filter`